src/rgb_led.rs

```diff
@@ -7,37 +7,48 @@ use nb::block;
 // Reference implementation:
 // https://github.com/smart-leds-rs/ws2812-spi-rs/blob/fac281eb57b5f72c48e368682645e3b0bd5b4b83/src/lib.rs
 
-const LED_COUNT: usize = 4;
+pub const LED_COUNT: usize = 4;
 const PI: f32 = 3.141_592_7e0;
 
 pub struct LedStrip<F: FullDuplex<u8>> {
     spi_bus: F,
 }
 
+#[derive(Copy, Clone)]
 pub struct Rgb {
```

Comment:
I think this struct might benefit from implementing some `Mul<f32>` and `Add<Rgb>` traits.

On top of that, it would be good to add an `r()`, `g()`, and `b()` function which do the float clamping and conversion to a `u8` for the caller.